spec/rails_helper.rb

```diff
@@ -54,4 +50,18 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+  config.before(:each) do
+    User.destroy_all
+    Bill.destroy_all
+  end
```

Comment:
How would the test database be cleared before each test? There is a uniqueness constraint on user emails, and each time the test suite is run, the same emails are attempted to be persisted, and the tests fail.

I can change this to `before(:suite)`, or the database_cleaner gem, but I think we need one or the other, no?